google_play_scraper/features/reviews.py

```diff
@@ -54,7 +56,9 @@ def reviews(
     country: str = "us",
     sort: Sort = Sort.NEWEST,
     count: int = 100,
+    device_id: int = None,
```

Comment:
When use this `device_id`?

Reply:
I thought of making `device_id` filtering empty by default.
We could then either not use it and no filtering on device would be applied

```
from google_play_scraper import Sort, reviews

result, continuation_token = reviews(
    'com.fantome.penguinisle',
    lang='en', # defaults to 'en'
    country='us', # defaults to 'us'
    sort=Sort.NEWEST, # defaults to Sort.NEWEST
    count=3, # defaults to 100
    filter_score_with=5 # defaults to None(means all score)
)
```

Or use it like this for example to filter on mobile only

```
from google_play_scraper import Device, Sort, reviews

result, continuation_token = reviews(
    'com.fantome.penguinisle',
    lang='en', # defaults to 'en'
    country='us', # defaults to 'us'
    sort=Sort.NEWEST, # defaults to Sort.NEWEST
    count=3, # defaults to 100
    filter_score_with=5 # defaults to None(means all score),
    filter_device_with=Device.MOBILE
)
```

Reply:
Readme could also be updated with such example btw

Reply:
It was defined as a functional factor, but there was no place to use it.

Reply:
Ah yes indeed my bad. For filtering on `device_id`, parameter to use in reviews definition is `filter_device_with` and not `device_id`. I now removed the duplicate parameter `device_id`. Sorry for the confusion